Common.props

```diff
@@ -28,7 +28,7 @@
     <!-- Reset $(BuildEpoch) whenever $(VersionPrefix) increments. We subtract this from YYYYMMDD portion of build
          number below to obtain the fourth part of file version that must fit in 16 bits. We can produce builds 
          for seven years from every epoch reset. -->
-    <VersionPrefix Condition="'$(VersionPrefix)' == ''">1.1.0</VersionPrefix>
+    <VersionPrefix Condition="'$(VersionPrefix)' == ''">2.0.0</VersionPrefix>
     <BuildEpoch>20170101</BuildEpoch>
```

Comment:
It should, but we're so close enough in time to current epoch, it doesn't really matter. (But moving forward, do as I say, not as I do ;))